src/metrics.rs

```diff
@@ -293,6 +293,8 @@ impl GooseRawRequest {
 /// knows which request is being updated.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct GooseRequestMetric {
+    // Time when the request was started.
+    pub request_time: u64,
```

Comment:
This helps us avoid gaps on the graph, but also (more importantly) gives us more accurate data. 

The only downside I can see is that we could get strange results if the time on various servers in the load test cluster would be out of sync.

Reply:
We can make a note in the Gaggle section of the book, "It's important to keep reliable time between your servers if you want accurate metrics." I don't see this as a downside or a concern, and it would already affect existing logging. NTP was created around 1980 ...

Reply:
This was removed.